settings.gradle

```diff
@@ -3,7 +3,13 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 import org.yaml.snakeyaml.Yaml
 
-includeBuild('tools/nimbus-gradle-plugin') {
+// We prefer `appServicesRootDir` over `rootDir` to help us on the path to the monorepo.
+// (They are the same in app-services but different in moz-central)
+def maybeAppServicesRootDir = new File(rootDir, "services/app-services")
```

Comment:
Some of this probably highlights my poor groovy/gradle skills, but I couldn't work out a sane way to share this variable both out and inside `setupProject`, so this is what I came up with.